packages/flutter_driver/lib/src/driver.dart

```diff
@@ -205,6 +208,61 @@ class FlutterDriver {
     return result.text;
   }
 
+  /// Starts recording performance traces.
+  Future<Null> startTracing() async {
+    try {
+      await _appIsolate.invokeExtension(_kStartTracingMethod, const {});
```

Comment:
`const {}` should be typed also (the lint doesn't yet check that but eventually will)